netbox/virtualization/api/serializers.py

```diff
@@ -84,14 +84,16 @@ class VirtualMachineSerializer(NetBoxModelSerializer):
 
     # Counter fields
     interface_count = serializers.IntegerField(read_only=True)
+    virtual_disk_count = serializers.IntegerField(read_only=True)
+    virtual_disk_space = serializers.IntegerField(read_only=True)
```

Comment:
I'm trying to decide what approach makes the most sense for migration here. I propose we:

1. Override the `disk` field on the serializer to return the value of the concrete `disk` field only if it's set, and otherwise return the aggregate size of all attached VirtualDisks.
2. Possibly also include a list of nested VirtualDisks, indicating the name and size of each. This would be a departure from the current pattern of not including child objects, but might make sense here.

Reply:
I didn't put in 2 as it was different then all other APIs and sort of a slipper slope.  Thought it might be better to have it like the other APIs and see if there are requests to add it directly to the main serializer.